migrations/1_initialize_schema.up.sql

```diff
@@ -481,6 +481,53 @@ CREATE TABLE issue_ticket (
   CONSTRAINT issue_ticket_pk PRIMARY KEY (issue_id, ticket_id)
 );
 
+----------------------------------------------------------------------------------------
+
+
+------------------------------ ACL Security --------------------------------------------
+
+create table acl_sid(
```

Comment:
Minor. Probably, it will be better to use Uppercase for uniformity.

Reply:
Fixed